tests/e2e/framework/kubernetes.go

```diff
@@ -248,6 +248,9 @@ func (k *KubeInfo) generateIstio(src, dst string) error {
 		content = []byte(strings.Replace(string(content), "LoadBalancer", "NodePort", 1))
 	}
 
+	content = []byte(strings.Replace(string(content), "args: [\"discovery\", \"-v\", \"2\"]",
```

Comment:
please use raw strings

`args: ["discovery", "-v", "2"]`, `args: ["discovery", "-v", "2", "-a", "`+k.Namespace+`"]`

Reply:
it'd be easier to read but that doesn't seem to matter too much and this has been crippling the infra, so if it passed andra's test and also passes the prow test, I'd say let's merge as is